pragma-node/src/constants/caches.rs

```diff
@@ -0,0 +1,13 @@
+// See:
+/// https://docs.rs/moka/latest/moka/future/struct.Cache.html#example-time-based-expirations
+
+/// Cache the update count for our onchain publishers because the query
+/// takes a lot of time.
+/// The cached value will live for the time specified in time to live.
+pub const PUBLISHERS_UDPATES_CACHE_TIME_TO_LIVE_IN_SECONDS: u64 = 20 * 60; // 20 minutes
+pub const PUBLISHERS_UDPATES_CACHE_TIME_TO_IDLE_IN_SECONDS: u64 = 5 * 60; // 5 minutes
+
+/// Cache of the stored Merkle Tree for a certain block in Redis.
+/// Since this value never change we can cache it for faster iterations.
+pub const MERKLE_FEED_TREE_CACHE_TIME_TO_LIVE_IN_SECONDS: u64 = 10 * 60; // 10 minutes
```

Comment:
can this cache be longer than the block time tho?

Reply:
It can but it's misleading probably, updated